pkg/imrpc/proxy.proto

```diff
@@ -32,6 +32,8 @@ service ProxyEngineService {
 	rpc SnapshotBackupStatus(EngineSnapshotBackupStatusRequest) returns (EngineSnapshotBackupStatusProxyResponse);
 	rpc BackupRestore(EngineBackupRestoreRequest) returns (EngineBackupRestoreProxyResponse);
 	rpc BackupRestoreStatus(ProxyEngineRequest) returns (EngineBackupRestoreStatusProxyResponse);
+	rpc CleanupAllMounts(google.protobuf.Empty) returns (google.protobuf.Empty);
```

Comment:
Suggest renaming to `CleanupBackupMountPoints`.

If we want to support cleaning up some mount points in the future, you could create a new request message with empty properties instead of using an empty message. This can make the method extensible, even though the current implementation will be just cleaning up all mount points. WDYT?

Reply:
Sounds good to me, thanks